.copyrightconfig

```diff
@@ -11,4 +11,4 @@ startyear: 2023
 #   - Dotfiles already skipped automatically
 # Enable by removing the leading '# ' from the next line and editing values.
 # filesexcluded: third_party/*, docs/generated/*.md, assets/*.png, scripts/temp_*.py, vendor/lib.js
-filesexcluded: .github/*, README.md, Jenkinsfile, gradle/*, docker-compose.yml, *.gradle, gradle.properties, gradlew, gradlew.bat
\ No newline at end of file
+filesexcluded: .github/*, README.md, CONTRIBUTING.md, Jenkinsfile, gradle/*, docker-compose.yml, *.gradle, gradle.properties, gradlew, gradlew.bat, *.json, *.xml, *.txt, **/test/resources/**, docs/**, flux-cli/hadoop/*
```

Comment:
Just do the "test/resources" and "docs" ones. Hopefully "test/resources" avoids the need for json/xml/txt . And remove "flux-cli/hadoop" as it's not relevant here.